### DISCUSSION TITLE
三个问题：关于区别笔记类别与inbound & outbound links的区别及作用

### DISCUSSION THREAD

**Discussion by lastsir** [windingwind/zotero-better-notes#1039] [Q&A] [answered]
三个问题：
1.如何区别笔记类别，因为之前对item notes和standalone notes的区别不是特别理解，建立一些笔记，但现在想弄清楚某个笔记到底是什么类型的笔记，不知道如何做。
2.更新升级后，可以看到standalone notes中右侧列显示inbound links & outbound links，尝试了一下，不是特别理解二者区别及作用场景。
3.在link creator这个窗口，有mention in和link to两个功能选项，也不是特别理解二者的区别及作用场景。

**Comment by windingwind** [comment, marked as answer]
1. item note即条目笔记，从属某个条目；standalone即不从属于条目。可在文库列表中看是否在某个条目的折叠树下。
2. inbound links：入链（其他笔记内有跳转当前笔记的链接，指向当前笔记）； outbound links：出链（当前笔记内有指向其他笔记的链接）
3. mention in：（在其他笔记中）提及，对应创建入链，将在其他笔记中创建一个指向当前笔记的链接；link to：指向（其他笔记），对应创建出链，将在当前笔记中创建一个指向其他笔记的链接。

说起来很绕，操作一下就明白了

**Reply by lastsir** [reply]
谢谢
后两个问题大概理解了
第一个问题是这样，有可能我把item notes从某个条目中移到了列表里了，所以现在我也不知道列表中的这几个notes到底是什么类型了

**Reply by windingwind** [reply]
是否是item note并不是笔记本身的属性，而是一个状态，从item下移出则就成为了standalone note。就好像就业，失业，状态是可以转换的。

**Reply by lastsir** [reply]
明白了，谢谢
